### ISSUE TITLE
minor  Ghidra throwing internal "ERROR" into user log file relative to gif/png

### ISSUE THREAD

**Issue by Hardcore-fs** [NationalSecurityAgency/ghidra#1102]
If you check out the  "ghidra user log", it is showing errors related to application images.

Start Ghidra, get in  project window ->help->show log.
log shows no errors
launch any project in the "Active project window " so the "dragon picture opens

close log window  & reopen
project window ->help->show log (in project window)

 - macOS 10.13.6 (17G8030)
 - Java Version: [e.g. 11.0.4]
 - Ghidra Version: [e.g. 9.0/9.1beta]

![ERROR](https://user-images.githubusercontent.com/1309004/65963962-6acfc100-e48e-11e9-8cf4-c1ea285bdb80.jpeg)

**Comment by astrelsky** [comment]
What does the data look like at those addresses?

**Comment by ryanmkurtz** [comment]
> What does the data look like at those addresses?

Yes, we'll need more info to have any hope of knowing what is going wrong.  Can you provide a sample binary that has the bad behavior?

**Comment by Hardcore-fs** [comment]
This is not a file I was actively looking at, just something dead I accidentally clicked on.

possibly if you went to this PUBLIC link:
http://81-223-137-237.static.upcbusiness.at/MAXDATA_Storage/Disk_Based_Storage/Archive/MAXDATA_SR1202/Firmware/SR1202_FirmwarePackage_2_4.zip

downloaded the file, Expanded & looked in :
"/storview embedded"
then looked at this 1.9mb file:
storview-3.10.0003-xyr-nb-en.bin

![broken1](https://user-images.githubusercontent.com/1309004/66004805-469ad100-e4dc-11e9-8a42-dd31e9cb9efe.jpeg)
